### ISSUE TITLE
windsave2table not reporting number of ionizing photons for macro-atom case

### ISSUE THREAD

**Issue by kslong** [sirocco-rt/sirocco#644]
The convergence ond other tables being produced are not indicating any ionizing photons in macro atom mode. For the simple atom case, there is no problem.  This suggests the issue somehow is when macro-atoms are involved.    This has been found in the tde models, so some more checking is required to make sure that the problem is not associated with these particular .pf files.

**Comment by saultyevil** [comment]
I think I may have figured this out about 30 minutes ago...

It looks like we only increment `xplasma->nioniz` in `radiation()`, which is not called in macro atom mode. Hence, nioniz can never incremented. I probably makes more sense to increment  nioniz in the same place where we calculate the ionisation parameter, in `update_banded_estimators()` [here](https://github.com/agnwinds/python/blob/82579fc6e1fba0771258c1aba77ec23868f6825a/source/radiation.c#L949), as this is called for both macro and non-macro case.

**Comment by saultyevil** [comment]
I've made the change I proposed above and plotted the number of ionising photons in the wind for our standard CV model.

![check](https://user-images.githubusercontent.com/19627279/69359719-ff7dc080-0c80-11ea-9be9-c7694b03d711.png)

The fact that the difference plot between the first two models is blank and that the macro atom now has ionising photons recorded is encouraging. I'll run a regression test overnight and create a PR to merge in if nothing falls over.
